models/staging/tpdm/base/base_tpdm__performance_evaluation.sql

```diff
@@ -0,0 +1,33 @@
+with educator_prep_programs as (
```

Comment:
CTE name isn't correct?

Reply:
Looks like copy paste typo. Please check in every model this issue

Reply:
agreed, this needs to change in all the files.

Reply:
thanks, guys, fixed